src/jit/rationalize.cpp

```diff
@@ -526,70 +503,54 @@ void copyFlags(GenTree *dst, GenTree *src, unsigned mask)
 
 //--------------------------------------------------------------------------------------
 // RewriteTopLevelComma - split a top-level comma into two top level statements.
-//                                returns (as out params) the two new locations
+//                                returns the location of the new statement.
 //--------------------------------------------------------------------------------------
```

Comment:
There is only one new statement: the current statement is reused and its tree replaced with the top-level comma's RHS. I'll clarify the comment.